cli/src/commands/install.js

```diff
@@ -100,7 +100,7 @@ export function setup(yargs: Yargs) {
       alias: 'o',
       describe: 'Overwrite an existing libdef',
       type: 'string',
-      demand: false
+      demand: false,
```

Comment:
I would prefer if you could revert this change, just to keep the history focused.

Reply:
No worries; dropped that commit.